### PR TITLE
1Password supports 2FA for Teams accounts using Duo Security

### PR DESCRIPTION
**Pull request by gingerbeardman**
See https://twitter.com/1Password/status/809091553381777409

### CODE REVIEW

**Review by mxxcon — CHANGES_REQUESTED** [review]
please don't include amazon change with this PR. please make it a separate PR.

**Comment by gingerbeardman** [comment]
Apologies, GitHub Desktop didn't warn me.

**Review by mxxcon — APPROVED** [review]

**Comment by mxxcon** [comment]
That's a pretty crappy setup of them. 😕 Use only 2fa service which requires payment to that service.

File: _data/retail.yml [hunk 1]
```diff
@@ -304,4 +304,3 @@ websites:
       tfa: No
       twitter: zazzle
       facebook: zazzle
-
```
Comment (RichJeanes) [inline _data/retail.yml:307]: Please do not remove new lines at the end of files. They should be there in all of our `.yml` files. See our `.editorconfig` for our formatting rules. If you use an editor that supports plugins, you can look for and install the Editor Config plug-in.

**Review by RichJeanes — CHANGES_REQUESTED** [review]

**Comment by gingerbeardman** [comment]
ok, thanks for the info.

changed

**Comment by Carlgo11** [comment]
I think that maybe we should wait for them to roll it out everywhere. From past experience they could very well disable 2FA again or never activate it for individuals.

**Comment by mxxcon** [comment]
@Carlgo11 but that's why there's exception text?

**Comment by Carlgo11** [comment]
@mxxcon yeah but then `tfa` is set to true.

**Review by RichJeanes — APPROVED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Only partially added. Don't think `tfa` should be set to `Yes` then.

**Review by jamcat22 — APPROVED** [review]
I say we should add it with the exception. We've taken this approach on other sites we've added, including Square, Dyn, Coinify, and arguably PayPal and Apple, whose 2FA systems don't protect their entire userbase and various sign in systems.

**Comment by gingerbeardman** [comment]
@jamcat22 @Carlgo11 FWIW I searched for other entries that use Duo Security and copied the way their entry was set up. Specifically, University of Michigan.

**Comment by Carlgo11** [comment]
@jamcat22 what if we put it as a `status` link then? To still put pressure on them.

**Comment by fpigerre** [comment]
A `status` link sounds like a good compromise, given that payment is required in order to use 2FA for 1Password.

**Comment by gingerbeardman** [comment]
redone as status link @Carlgo11 @psgs 

still think exception is better as does @jamcat22

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**